From ed5b88e6079b11b5f33b0795a5254e8e9c5f6e54 Mon Sep 17 00:00:00 2001
From: Boris Ranto <branto@redhat.com>
Date: Fri, 22 Apr 2016 15:30:22 +0200
Subject: [PATCH] Do not use -momit-leaf-frame-pointer flag

The -momit-leaf-frame-pointer flag that rocksdb is being built with does
not exist on armv7hl, this patch disables the flag.

Signed-off-by: Boris Ranto <branto@redhat.com>
---
 src/rocksdb/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/rocksdb/Makefile b/src/rocksdb/Makefile
index c64ea36..b3758a5 100644
--- a/src/rocksdb/Makefile
+++ b/src/rocksdb/Makefile
@@ -84,9 +84,6 @@ endif
 # compile with -O2 if debug level is not 2
 ifneq ($(DEBUG_LEVEL), 2)
 OPT += -O2 -fno-omit-frame-pointer
-ifneq ($(MACHINE),ppc64) # ppc64 doesn't support -momit-leaf-frame-pointer
-OPT += -momit-leaf-frame-pointer
-endif
 endif
 
 # if we're compiling for release, compile without debug code (-DNDEBUG) and
-- 
2.5.5

